pyvista/plotting/render_window_interactor.py

```diff
@@ -74,7 +74,7 @@ def remove_observer(self, event):
 
     def remove_observers(self):
         for obs in self._observers.values():
-            self.interactor.RemoveObservers(obs)
+            self.remove_observer(obs)
```

Comment:
There's a chance that this will raise because the `self._observers` dict would change size during iteration. You might have to copy the `values()` by creating a list from them.

Reply:
Good point, I missed that completely :+1:

Reply:
Sorry, code was already changed by the time I made this comment.